{
  "id": "oval:org.opensuse.security:def:202331047",
  "version": "1",
  "class": "vulnerability",
  "metadata": {
    "Title": "CVE-2023-31047",
    "affected": {
      "family": "unix",
      "Platform": [
        "openSUSE Leap 15.5"
      ]
    },
    "reference": [
      {
        "ref_id": "Mitre CVE-2023-31047",
        "ref_url": "https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2023-31047",
        "source": "CVE"
      },
      {
        "ref_id": "SUSE CVE-2023-31047",
        "ref_url": "https://www.suse.com/security/cve/CVE-2023-31047",
        "source": "SUSE CVE"
      },
      {
        "ref_id": "SUSE-SU-2023:2839-1",
        "ref_url": "https://lists.suse.com/pipermail/sle-security-updates/2023-July/015497.html",
        "source": "SUSE-SU"
      }
    ],
    "Description": "\n    In Django 3.2 before 3.2.19, 4.x before 4.1.9, and 4.2 before 4.2.1, it was possible to bypass validation when using one form field to upload multiple files. This multiple upload has never been supported by forms.FileField or forms.ImageField (only the last uploaded file was validated). However, Django's \"Uploading multiple files\" documentation suggested otherwise.\n    ",
    "advisory": {
      "from": "security@suse.de",
      "Severity": "Moderate",
      "cve": {
        "text": "CVE-2023-31047",
        "href": "https://www.suse.com/security/cve/CVE-2023-31047/",
        "impact": "moderate",
        "cvss3": "5.6/CVSS:3.1/AV:N/AC:H/PR:N/UI:N/S:U/C:L/I:L/A:L"
      },
      "bugzilla": [
        {
          "text": "SUSE bug 1210866",
          "href": "https://bugzilla.suse.com/1210866"
        }
      ],
      "issued": {
        "date": "2023-07-15"
      },
      "updated": {
        "date": "2023-07-15"
      },
      "affected_cpe_list": {
        "Cpe": [
          "cpe:/o:opensuse:leap:15.5"
        ]
      }
    }
  },
  "criteria": {
    "operator": "AND",
    "criterias": [
      {
        "operator": "AND",
        "criterions": [
          {
            "test_ref": "oval:org.opensuse.security:tst:2009771998",
            "comment": "python3-Django-2.0.7-150000.1.11.1 is installed"
          },
          {
            "test_ref": "oval:org.opensuse.security:tst:2009173965",
            "comment": "python3-Django is signed with openSUSE key"
          }
        ]
      }
    ],
    "criterions": [
      {
        "test_ref": "oval:org.opensuse.security:tst:2009752682",
        "comment": "openSUSE Leap 15.5 is installed"
      }
    ]
  }
}
